xlators/mgmt/glusterd/src/glusterd-store.c

```diff
@@ -4387,7 +4391,10 @@ glusterd_store_peerinfo_uuid_shandle_create(glusterd_peerinfo_t *peerinfo)
     char peerfpath[PATH_MAX];
     int32_t ret = -1;
 
-    glusterd_store_uuid_peerpath_set(peerinfo, peerfpath, sizeof(peerfpath));
+    if (glusterd_store_uuid_peerpath_set(peerinfo, peerfpath,
+                                         sizeof(peerfpath)) < 0)
+        return ret;
+
```

Comment:
Better to save the return value of the function call in `ret` and then check the value of `ret` and return it. The reason is if in the future some piece of code is added above it which updates the `ret` value, then he/she might not update this check accordingly.
```
ret = glusterd_store_uuid_peerpath_set(peerinfo, peerfpath, sizeof(peerfpath)));
if (ret)
    return -1;
```